### PR TITLE
added row_stack pytorch frontend function

### LINKED ISSUE #10154: row_stack

### PR DESCRIPTION
**Pull request by adhi1910**
Close #10154

### CODE REVIEW

**Review by guillesanbri — CHANGES_REQUESTED** [review]
Hi! Thank you very much for the contribution! I have requested some changes, let me know if I can help with anything :slightly_smiling_face:

File: ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py [hunk 1]
```diff
@@ -977,3 +977,29 @@ def test_torch_dsplit(
         input=value[0],
         indices_or_sections=indices_or_sections,
     )
+
+
+# row_stack
+@handle_frontend_test(
+    fn_tree="torch.row_stack",
+    dtype_value_shape=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```
Comment (guillesanbri) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py:986]: the test seems to work fine, but you should add `num_arrays=st.integers(1, 5),` here, as currently we are only passing one array and therefore not testing the function properly :slightly_smiling_face:

**Review by guillesanbri — CHANGES_REQUESTED** [review]
just some final changes :grin: btw, I think that the problem with the files in the idea folder is that the files didn't have a newline at the end of the file

**Comment by adhi1910** [comment]
@guillesanbri I have added the requested changes and for the idea file new line(eof) has been added. But it still shows an error. Can you please look into it? and BTW sorry for more commits (all are reformating issues).

**Comment by guillesanbri** [comment]
don't worry about the commits, it's squashed into one before merging so no problem haha. I have fixed the `.idea` files and merged your PR, thank you very much for the contribution!

P.S. I don't really know why the EOF thing was happening, but for future reference, it's best to just add and commit changes to the files that you have modified :slightly_smiling_face:

**Pull request merged**